addons/payment/wizards/payment_link_wizard_views.xml

```diff
@@ -13,8 +13,11 @@
                         <field name="partner_id" invisible="1"/>
                         <field name="partner_email" invisible="1"/>
                         <field name="amount_max" invisible="1"/>
+                        <field name="available_acquirer_ids" invisible="1"/>
+                        <field name="has_multiple_acquirers" invisible="1"/>
                         <field name="description"/>
                         <field name="amount"/>
+                        <field name="acquirer_id" help="Let blank to select all acquirers" placeholder="Let blank to select all acquirers" options="{'no_open': True, 'no_create': True}" attrs="{'invisible':[('has_multiple_acquirers', '=', False)]}"/>
```

Comment:
Specify the help on the python field, not in the view ;)
And do not hesitate to indent xml lines when they are long (overly long lines are a nightmare in python but also in xml).